scripts/ethcommands.ts

```diff
@@ -184,6 +186,56 @@ export const bridgeNativeTokenToL3Command = {
   },
 };
 
+export const transferL3ChainOwnershipCommand = {
+  command: "transfer-l3-chain-ownership",
+  describe: "transfer L3 chain ownership to upgrade executor",
+  builder: {
+    creator: {
+      string: true,
+      describe: "address of the token bridge creator",
+    },
+    wait: {
+      boolean: true,
+      describe: "wait till ownership is transferred",
+      default: false,
+    },
```

Comment:
is this unused

Reply:
It is used in `sendTransaction` function (waits for receipt)